polynote-frontend/style/styles.less

```diff
@@ -304,12 +310,13 @@ body {
       bottom: 0;
       width: @cell-margin-width;
       background: @ui-background;
-      border: 1px solid @ui-border;
+      border-top-left-radius: 10px;
```

Comment:
Not sure I agree with the new border-radii. They look a bit odd to me when the cell is active. I'd argue for the clean corners.